### ISSUE TITLE
Twitter output node gives 401

### ISSUE THREAD

**Issue by stevegg** [node-red/node-red-nodes#174]
I had posted this issue in the node-red repo and will remove it from there.

I have a Twitter output node that throws a 401 in the debug tab whenever I attempt to send a payload. I have authenticated it properly (several times) but get this:

Error: HTTP Error 401: Unauthorized, API message: {"errors":[{"code":32,"message":"Could not authenticate you."}]}

I'm sure this used to work fine.

Time Zone/Date & Time are setup properly on node red host and OAuth completes successfully (I see this at the end of the OAuth flow: Authorised - you can close this window and return to Node-RED)

What am I missing?

**Comment by andypiper** [comment]
Are you able to post to Twitter successfully using the same account, but not using node-red?

**Comment by stevegg** [comment]
Yes, I can post fine with the same account...just not using the twitter node in Node Red.

**Comment by jeffoxenberg** [comment]
I stumbled across this because I experienced the same issue, and was just able to solve it.  I kept getting the same 401 error, and realized that I was sending a JSON object to the twitter node instead of a string.  I changed the message payload to a string and it works now.  YMMV.
